zerver/lib/events.py

```diff
@@ -331,11 +330,6 @@ def fetch_initial_state_data(
         state["realm_edit_topic_policy"] = (
             EditTopicPolicyEnum.ADMINS_ONLY if user_profile is None else realm.edit_topic_policy
         )
-        state["realm_delete_own_message_policy"] = (
-            CommonMessagePolicyEnum.ADMINS_ONLY
-            if user_profile is None
-            else realm.delete_own_message_policy
-        )
```

Comment:
`CommonMessagePolicyEnum` is now unused and can be deleted.

Reply:
It is used in `zerver/migrations/0001_squashed_0569.py`.

Reply:
OK, we can move it into that migration file, then. I'll add a commit to manage this.